pkg/client/rootd/service.go

```diff
@@ -352,6 +358,14 @@ func run(c context.Context, loggingDir, configDir string) error {
 		return err
 	}
 
+	tracer, err := tracing.NewTraceServer(c, tracing.TraceConfig{
+		ProcessID:   4,
```

Comment:
is this not supposed to be the OS PID?

Reply:
you're totally right, I didn't think this through; there's also a whole bunch of other resource stuff we can pass through, I'm gonna refactor this bit